### ISSUE TITLE
Uploaded dataset seems incomplete?

### ISSUE THREAD

**Issue by ShadowTinker** [Applied-Machine-Learning-Lab/PLATE#2]
Hi, @wyhwhy, Thanks for your innovative work!

When I try to run the pre-training script following the README file `python douban_main_all.py --model_name pdfm_user_autodis  --learning_rate 2e-3 --job 1`, it turns out that perhaps some raw data file named 'ratings.dat' is missing.
<img width="953" alt="image" src="https://github.com/Applied-Machine-Learning-Lab/PLATE/assets/63909836/95855609-d9ad-4d89-9a0b-5ef939103ff4">

And the provided datasets have already been processed. So I’d be grateful if you could teach me how to run the code given the 9 splitted datasets.

**Comment by wyhwhy** [comment]
Thanks for your reminder and I will fix it soon. Actually all the 3 domains (music, book, movie) are split into train/valid/test set using random seed 0, so there are 9 csv files.

**Comment by wyhwhy** [comment]
> Hi, @wyhwhy, Thanks for your innovative work!
> 
> When I try to run the pre-training script following the README file `python douban_main_all.py --model_name pdfm_user_autodis --learning_rate 2e-3 --job 1`, it turns out that perhaps some raw data file named 'ratings.dat' is missing. <img alt="image" width="953" src="https://user-images.githubusercontent.com/63909836/262693552-95855609-d9ad-4d89-9a0b-5ef939103ff4.png">
> 
> And the provided datasets have already been processed. So I’d be grateful if you could teach me how to run the code given the 9 splitted datasets.

Hi, I think I have just fixed it. Please clone the code and try again.

**Comment by ShadowTinker** [comment]
Thank you for your timely reply @wyhwhy!

I've tried again and successfully reproduced the results in the paper by making some additional changes to the code:
1. https://github.com/Applied-Machine-Learning-Lab/PLATE/blob/1bb58652548f65276646a4f471648d1f7d6911fb/Douban_code/douban_main_all.py#L154 to `parser.add_argument('--save_dir', default='.')`
2. https://github.com/Applied-Machine-Learning-Lab/PLATE/blob/1bb58652548f65276646a4f471648d1f7d6911fb/Douban_code/dataset/douban.py#L5 to `class Douban(torch.utils.data.Dataset):`. Similar changes are applied to the other 3 dataset classes.
3. https://github.com/Applied-Machine-Learning-Lab/PLATE/blob/1bb58652548f65276646a4f471648d1f7d6911fb/Douban_code/dataset/douban.py#L16 to `def __init__(self, mode, path_base='./dataset/'):`. Similar changes are applied to the other 3 dataset classes.

Thanks again for your kind reply and intriguing work.

**Comment by wyhwhy** [comment]
> > Hi, @wyhwhy, Thanks for your innovative work!
> > When I try to run the pre-training script following the README file `python douban_main_all.py --model_name pdfm_user_autodis --learning_rate 2e-3 --job 1`, it turns out that perhaps some raw data file named 'ratings.dat' is missing. <img alt="image" width="953" src="https://user-images.githubusercontent.com/63909836/262693552-95855609-d9ad-4d89-9a0b-5ef939103ff4.png">
> > And the provided datasets have already been processed. So I’d be grateful if you could teach me how to run the code given the 9 splitted datasets.
> 
> Hi, I think I have just fixed it. Please clone the code and try again.

Thanks for supporting our work and pointing out these bugs! I just fixed the new ones mentioned.

**Comment by wyhwhy** [comment]
> Thank you for your timely reply @wyhwhy!
> 
> I've tried again and successfully reproduced the results in the paper by making some additional changes to the code:
> 
> 1. https://github.com/Applied-Machine-Learning-Lab/PLATE/blob/1bb58652548f65276646a4f471648d1f7d6911fb/Douban_code/douban_main_all.py#L154
>     to `parser.add_argument('--save_dir', default='.')`
> 2. https://github.com/Applied-Machine-Learning-Lab/PLATE/blob/1bb58652548f65276646a4f471648d1f7d6911fb/Douban_code/dataset/douban.py#L5
>     to `class Douban(torch.utils.data.Dataset):`. Similar changes are applied to the other 3 dataset classes.
> 3. https://github.com/Applied-Machine-Learning-Lab/PLATE/blob/1bb58652548f65276646a4f471648d1f7d6911fb/Douban_code/dataset/douban.py#L16
>     to `def __init__(self, mode, path_base='./dataset/'):`. Similar changes are applied to the other 3 dataset classes.
> 
> Thanks again for your kind reply and intriguing work.

Thanks for supporting our work and pointing out these bugs! I just fixed the new ones mentioned.
